gamemodes/terrortown/gamemode/shared/hud_elements/tttpickup/pure_skin_pickup.lua

```diff
@@ -84,7 +76,7 @@ if CLIENT then
 
 		if item.type == PICKUP_WEAPON then
 			tipColor = LocalPlayer():GetRoleColor()
-			icon = self.SlotIcons[item.kind] or self.icon_item
+			icon = BaseClass.SlotIcons[item.kind] or self.icon_item
```

Comment:
```suggestion
			icon = self.SlotIcons[item.kind] or self.icon_item
```

This should work since it is inherited from the base